### ISSUE TITLE
TLS config for Redis client

### ISSUE THREAD

**Issue by vranystepan** [dapr/components-contrib#95]
## Describe the proposal
Hello, I'm trying to use Dapr with Digital Ocean managed Redis which is using secured connection. I'm getting following message:

```
time="2019-11-02T17:45:45Z" level=warning msg="error initializing pub sub pubsub.redis: redis streams: error connecting to redis at private-dapr01-do-user-2603541-0.db.ondigitalocean.com:25061: EOF"
```

I suspect it is caused by the missing TLS configuration here:

https://github.com/dapr/components-contrib/blob/master/pubsub/redis/redis.go#L69

```go
	client := redis.NewClient(&redis.Options{
		Addr:            m.host,
		Password:        m.password,
		DB:              0,
		MaxRetries:      3,
		MaxRetryBackoff: time.Second * 2,
	})
```

Could you confirm or refute whether the current implementation of Redis pub sub supports secure connection?

Thanks in advance!

**Comment by yaron2** [comment]
Hi @vranystepan, you are correct.

This issue will be resolved via https://github.com/dapr/components-contrib/pull/96.

**Comment by vranystepan** [comment]
> Hi @vranystepan, you are correct.
> 
> This issue will be resolved via #96.

Great, thank you so much! ❤️
